kie-config-cli/Readme.md

```diff
@@ -27,12 +27,6 @@ Available commands
 * remove-repo - remove existing repository from config only
 * add-repo-org-unit - add repository to the organizational unit
 * remove-repo-org-unit - remove repository from the organizational unit
-* add-role-repo - add role(s) to repository
```

Comment:
the api has changed, but I (and probably our customers) see the need of a CLI tool to manage ACL